### DISCUSSION TITLE
Welcome to genius-ai Discussions!

### DISCUSSION THREAD

**Discussion by sanidhyy** [sanidhyy/genius-ai#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SuhasDhawale** [comment]
Hello sanidhya , I am getting some errors while I'm trying to build this I really wish if you could help me just let me know and we'll share contacts and have a little chat for better understanding

**Reply by sanidhyy** [reply]
Hey! No worries, I’m here to help. Just hit me up on Discord—my ID is "sanidhyy." We can chat there and sort everything out. Looking forward to it! 😊

**Reply by SuhasDhawale** [reply]
I sent you a friend request there, my username is catheris
